.github/workflows/package-build.yaml

```diff
@@ -0,0 +1,40 @@
+name: Mautic language packs
+on:
+    schedule:
+        -   cron: '0 * * * *'
```

Comment:
Reading about the schedule:

> Scheduled workflows run on the latest commit on the default or base branch.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

So I guess we'll have to merge this for it to start.

Reply:
But maybe you can try the classic 
```
on:
  pull_request:
```
instead just to test if it works on this PR and then return the schedule there before merge?

Reply:
This worked.